src/reference/02-DetailTopics/01-Using-sbt/03-Cross-Build.md

```diff
@@ -341,6 +341,23 @@ than the default.
   }
 }
 ```
+### Scala 3 specific cross-versions
```

Comment:
Thanks! Since Scala 3 cross-building is likely to be used more than the other CrossVersion stuff, maybe this should be near the top of page, or maybe somewhere at the top of the page there can be a link to this section?

Reply:
In the release note I added a chunky warning with `CrossVersion.for3Use2_13`. I think we should bring that in as well since it's not risk-free to do this.

Reply:
Agree, though I think that warning as-is looks more scary than it is: it sounds like application developers need to carefully inspect their transitive dependencies to avoid mixing _2.13 and _3, but actually sbt detects these conflicts itself and emit an error, so it shouldn't be possible to accidentally end up with both versions on the classpath.

Reply:
Not sure what % is app developers vs what % is library authors, but generally it's safe for application developers (or in-house library authors), but we should encourage library authors to cross build against 3.0 universe when possible and avoid tangling 2.13 and/or 3.x back and forth (as djspiewak advocates). Generally there are more app developers than library authors, but if you add "someone who is reading sbt cross building docs" as a prior, maybe the percentage is 50:50?